src/Explorer/Notebook/NotebookComponent/__mocks__/rx-jupyter.ts

```diff
@@ -1,6 +1,6 @@
 import { Observable, of } from "rxjs";
 import { AjaxResponse } from "rxjs/ajax";
-import { ServerConfig } from "rx-jupyter";
+import { ServerConfig } from "@nteract/types";
```

Comment:
Check with Laurent

Reply:
Hi, Laurent, if we "import { ServerConfig } from "rx-jupyter"; we will meet TS2459 error

Reply:
@languy

Reply:
The full error is: `src/Explorer/Notebook/NotebookComponent/__mocks__/rx-jupyter.ts(3,10): error TS2459: Module '"../../../../../node_modules/rx-jupyter/lib"' declares 'ServerConfig' locally, but it is not exported.`

It's mentioned in line 14: ```typescript
export const sessions = {
  create: (serverConfig: ServerConfig, body: object): Observable<AjaxResponse> => of(fakeAjaxResponse),
  __setResponse: (response: AjaxResponse) => {
    fakeAjaxResponse = response;
  },
  createSpy: undefined as any
};
```
The actual `serverConfig` is never used, so typescript probably doesn't export it.
Replace with: `create(serverConfig: unknown, body: object)` so you don't have to worry about it.

Reply:
Ok, thanks